ksy_schema.json

```diff
@@ -117,7 +124,8 @@
               "type": "array",
               "items": { "type": "string" }
             }
-          ]
+          ],
+          "description": "name(s) of file extensions, usually associated with this format"
```

Comment:
I think the "name(s)" part isn't needed - for example .ksy is a file extension, not the *name of* a file extension.

I would say "used for this format", not "associated with this format". The way I know it, file extensions are "associated with" an *application*, not a format - for example the extension .bmp is used for Windows bitmap images, but is *associated with* MS Paint.

I would also mention that the extension(s) listed here shouldn't include a dot at the start (you have to write `file-extension: ksy`, not `file-extension: .ksy`).